docs/pages/connect-your-client/request-access.mdx

```diff
@@ -0,0 +1,302 @@
+
+```code
+$ kubectl api-resources --namespaced=true -o name --sort-by=name
+```
+</Admonition>
+
+For example, to request access to a pod called `nginx-1` in the `dev`
+namespace, run the following command:
+
+```code
+$ tsh request create --resource /teleport.example.com/kube:ns:pods/mycluster/dev/nginx-1
+```
+
+And to request access to a deployment called `website` in the `prod`
+namespace, run the following command:
+
+```code
+$ tsh request create --resource /teleport.example.com/kube:ns:deployments.apps/mycluster/prod/website
+```
+
+For the `NAMESPACED_PLURAL_KIND` value you can use `*` to match all.
+For the `API_GROUP`, `NAMESPACE` and `RESOURCE_NAME`
+values, you can match ranges of characters by supplying a wildcard (`*`) or
+regular expression. Regular expressions must begin with `^` and end with `$`.
+
+For example, to create a request to access all pods in all namespaces that
+match the regular expression `/^nginx-[a-z0-9-]+$/`, run the following command:
+
+```code
+$ tsh request create --resource '/teleport.example.com/kube:ns:pods/mycluster/*/^nginx-[a-z0-9-]+$'
+```
+
+### Cluster-scoped resources
+
+For Kubernetes cluster-scoped resources, the `resource-id` is in the following
+format:
+
+```
+/TELEPORT_CLUSTER/kube:cw:CLUSTER_WIDE_PLURAL_KIND[.API_GROUP]/KUBE_CLUSTER/RESOURCE_NAME
+```
+
+The `API_GROUP` part is only required for resources that are part of the core
+kubernetes group.
+
+<Admonition type="tip">
+You can run this command to list the cluster-wide Kubernetes resources,
+including their API groups:
+
+```code
+$ kubectl api-resources --namespaced=false -o name --sort-by=name
+```
+</Admonition>
+
+For example, to request access to a namespace called `prod`, run the
+following command:
+
+```code
+$ tsh request create --resource /teleport.example.com/kube:cw:namespaces/mycluster/prod
+```
+
+Note that this will only grant access to the namespace resource itself, not to
+any namespaced resources within it.
+
+For the `CLUSTER_WIDE_PLURAL_KIND` value you can use `*` to match all.
+For the `API_GROUP` and `RESOURCE_NAME` value, you
+can match ranges of characters by supplying a wildcard (`*`) or regular
+expression. Regular expressions must begin with `^` and end with `$`.
+
+For example, to create a request to access all namespaces prefixed with `dev` match
+the regular expression `/^dev-[a-z0-9-]+$/`, run the following command:
+
+```code
+$ tsh request create --resource '/teleport.example.com/kube:cw:namespaces/mycluster/^dev-[a-z0-9-]+$'
+```
+
+Using cluster-scoped resource requests requires the role under `search_as_roles` to have the correct
+permissions for the cluster-scoped resource.  This is an example of the required permissions to
+request access only to the namespace `prod` (does not include any resources within it):
+
+```yaml
+kind: role
+version: v8
+spec:
+  allow:
+    kubernetes_resources:
+    - kind: namespaces
+      api_group: ''
+      name: prod
+      verbs:
+      - '*'
+```
```

Comment:
If we don't want end users to see yaml, should we remove this?

Reply:
Yes, I'll remove it.